docs/contributing/code-style.md

```diff
@@ -107,15 +107,6 @@ some_objs = UserProfile.objects.get(id=17)
 assert obj.id in set([o.id for i in some_objs])
 ```
 
-### user\_profile.save()
-
-You should always pass the update\_fields keyword argument to .save()
-when modifying an existing Django model object. By default, .save() will
-overwrite every value in the column, which results in lots of race
-conditions where unrelated changes made by one thread can be
-accidentally overwritten by another thread that fetched its UserProfile
-object before the first thread wrote out its change.
```

Comment:
Hmm, the `user_profile` version is handled by lint, but not the similar ones for `stream`, `realm`, etc.  So maybe we should change this to `realm` for now?  

I think with `semgrep`, we could in theory enforce this via lint on all common objects.  @amanagr FYI.

Reply:
(The reason `semgrep` is relevant is if the local variable isn't called `user_profile`, the current linter rule won't work)

Reply:
Yeah correct. I will try to implement this with semgrep.